src/org/openstreetmap/josm/plugins/osmobjinfo/OSMObjInfoActions.java

```diff
@@ -39,17 +52,17 @@ public static void openinBrowserUserNeis(String user) {
 
     static void openinBrowserUserOsmComments(String user) {
         if (!user.isEmpty()) {
-            openInBrowser( "https://www.mapbox.com/osm-comments/#/changesets/?q=users:".concat(user));
+            openInBrowser("https://www.mapbox.com/osm-comments/#/changesets/?q=users:".concat(user));
         }
     }
 
     public static void copyChangeset(String idChangeset) {
         if (!idChangeset.isEmpty()) {
-            String linkchangeset = "https://www.openstreetmap.org/changeset/" + idChangeset;
+            String linkchangeset = "https://www.openstreetmap.org/changeset/".concat(idChangeset);
```

Comment:
Config.getUrls().getBaseBrowseUrl() should be used instead of hardcoding https://www.openstreetmap.org

Reply:
Fair enough. I think I was just reusing the pre-existing code.